### PR TITLE
Updating twig required version (fixing vulnerability) 4.4

### PR DESCRIPTION
**Pull request by ferjep**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

https://symfony.com/blog/twig-security-release-disallow-non-closures-in-the-sort-filter

### CODE REVIEW

File: src/Symfony/Bundle/TwigBundle/composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "symfony/http-kernel": "^4.4",
         "symfony/polyfill-ctype": "~1.8",
         "symfony/polyfill-php80": "^1.16",
-        "twig/twig": "^1.43|^2.13|^3.0.4"
+        "twig/twig": "^1.43|^2.14|^3.3.8"
```
Comment (GromNaN) [inline src/Symfony/Bundle/TwigBundle/composer.json]: Should include the bugfix version.
```suggestion
        "twig/twig": "^1.43|^2.14.11|^3.3.8"
```

**Review by GromNaN — CHANGES_REQUESTED** [review]
Can you update the constraint in other places where `twig/twig` is required.

- composer.json
- src/Symfony/Bridge/Twig/composer.json
- src/Symfony/Bundle/FrameworkBundle/composer.json
- src/Symfony/Bundle/SecurityBundle/composer.json
- src/Symfony/Component/HttpKernel/composer.json
- src/Symfony/Component/VarDumper/composer.json

File: src/Symfony/Bundle/TwigBundle/composer.json (hunk 1 above)
Comment (ferjep) [reply src/Symfony/Bundle/TwigBundle/composer.json]: Done, changed in

- composer.json
- src/Symfony/Bridge/Twig/composer.json
- src/Symfony/Bundle/FrameworkBundle/composer.json
- src/Symfony/Bundle/SecurityBundle/composer.json
- src/Symfony/Bundle/TwigBundle/composer.json
- src/Symfony/Bundle/WebProfilerBundle/composer.json
- src/Symfony/Component/HttpKernel/composer.json
- src/Symfony/Component/VarDumper/composer.json

**Review by GromNaN — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
:-1:, it's not our responsibility to bump deps, even for security issues.

**Comment by fabpot** [comment]
I agree, also because Composer will always install the maximum possible version allowed by the constraint when creating a new project (or when updating dependencies).

**Pull request closed without merging**